README.md

```diff
@@ -6,23 +6,12 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 ## Basic requirements
 
-### xcode
-
-1.  Install xcode from app store
-1.  Open xcode and agree to license and finish the installation
-
 ### Install node.js and npm with [nvm](https://github.com/creationix/nvm)
```

Comment:
I'd suggest simplifying this to point say something like:

> **Install Node and NPM**
> The project requires node v14 or later. Follow installation instructions on [nodejs.org](https://nodejs.org/en/). 

`nvm` is not necessary to run the project, and I think this section makes it look more complicated than it really is to get started.

Reply:
I previously had issues running it on newer node so will say `node@14` for now.